examples/complete/main.tf

```diff
@@ -0,0 +1,50 @@
+module "atlas_basic" {
+  source = "../../"
+  org_id = var.org_id
+
+  project_name = "complete-example-project"
+  ip_addresses = ["1.2.3.4", "6.7.8.9"]
+  cidr_blocks  = ["10.1.0.0/16", "12.2.0.0/16"]
+
+  database_users = [
+    {
+      username = "user1"
+      password = "1234"
```

Comment:
These are weak passwords. I would consider using the `random_password` resource here.